src/ffi/mod.rs

```diff
@@ -681,9 +733,9 @@ pub extern "C" fn telio_disconnect_from_exit_nodes(dev: &telio) -> telio_result
 /// # Parameters
 /// - `cfg`: Output of GET /v1/meshnet/machines/{machineIdentifier}/map
 ///
-pub extern "C" fn telio_set_meshnet(dev: &telio, cfg: *const c_char) -> telio_result {
+pub extern "C" fn telio_set_meshnet(telio: &telio, cfg: *const c_char) -> telio_result {
```

Comment:
Nit: I think this should go back to `dev`. The `&telio` param in all other functions here is called `dev`, I think we should be consistent, and since we're (finally) getting UniFFI soon, it makes sense to leave as much as possible the same